src/Bridge/Symfony/Bundle/Action/DocAction.php

```diff
@@ -100,7 +102,14 @@ public function __invoke(Request $request)
 
         $documentation = new Documentation($this->resourceNameCollectionFactory->create(), $this->title, $this->description, $this->version, $this->formats);
 
-        return new Response($this->twig->render('@ApiPlatform/SwaggerUi/index.html.twig', $this->getContext($request, $documentation)));
+        if (DocListener::GRAPHQL === $request->get('ui')) {
```

Comment:
You should store the result of `$request->get('ui')` in a variable (calling a PHP function costs a bit)